Index: src/main/java/com/demo/filesystem/model/BlockTable.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/demo/filesystem/model/BlockTable.java b/src/main/java/com/demo/myfilesystem/model/BlockTable.java
rename from src/main/java/com/demo/filesystem/model/BlockTable.java
rename to src/main/java/com/demo/myfilesystem/model/BlockTable.java
--- a/src/main/java/com/demo/filesystem/model/BlockTable.java	(revision d86c54fb896d76304b79b6835709e35a2a4b462f)
+++ b/src/main/java/com/demo/myfilesystem/model/BlockTable.java	(date 1696268235724)
@@ -1,4 +1,4 @@
-package com.demo.filesystem.model;
+package com.demo.myfilesystem.model;
 
 import javafx.geometry.Insets;
 import javafx.geometry.Pos;
@@ -6,17 +6,14 @@
 import javafx.scene.paint.Color;
 import javafx.scene.shape.Rectangle;
 
-import java.lang.reflect.Array;
-import java.util.ArrayList;
-
-import static com.demo.filesystem.utils.Constant.blocksNumOfDisk;
+import static com.demo.myfilesystem.utils.Constant.BLOCKS_NUM_OF_DISK;
 
 /**
  * 右上侧磁盘各个块占用情况
  */
 public class BlockTable extends GridPane {
     private final static int COLS = 8;
-    private final static int ROWS = blocksNumOfDisk / COLS; // 一行8个
+    private final static int ROWS = BLOCKS_NUM_OF_DISK / COLS; // 一行8个
     private final static int REC_WIDTH = 15;
     private final static int REC_HEIGHT = 10;
 
